integration/integration_test.go

```diff
@@ -183,7 +184,6 @@ func TestRun(t *testing.T) {
 
 func TestGitBuildcontext(t *testing.T) {
 	repo := "github.com/GoogleContainerTools/kaniko"
-	dockerfile := "integration/dockerfiles/Dockerfile_test_run_2"
```

Comment:
This causing the integration tests to fail as the references to `dockerfile` have not been updated 

```
# github.com/GoogleContainerTools/kaniko/integration [github.com/GoogleContainerTools/kaniko/integration.test]
./integration_test.go:193:10: undefined: dockerfile
./integration_test.go:206:10: undefined: dockerfile
./integration_test.go:248:10: undefined: dockerfile
./integration_test.go:286:2: undefined: dockerfile
./integration_test.go:291:10: undefined: dockerfile
```